docker-start.sh

```diff
@@ -0,0 +1,15 @@
+if [[ ${farmer} == 'true' ]]; then
+  chia start farmer-only
+elif [[ ${harvester} == 'true' ]]; then
+  if [[ -z ${farmer_address} || -z ${farmer_port} || -z ${ca} ]]; then
+    echo "A farmer peer address, port, and ca path are required."
+    exit
+  else
+    chia configure --set-farmer-peer ${farmer_address}:${farmer_port}
+    chia start harvester
+  fi
+else
+  chia start farmer
+fi
+
+tail -f ~/.chia/mainnet/log/debug.log
```

Comment:
```suggestion
tail -f $CHIA_ROOT/log/debug.log
```

This file doesn't necessarily exist right away either - I just tested a build of this locally (with latest main merged in) and I'm getting:

```
...
Daemon not started yet
Starting daemon
chia_harvester: started
chia_farmer: started
chia_full_node: started
chia_wallet: started
tail: cannot open '/root/.chia/mainnet/log/debug.log' for reading: No such file or directory
tail: no files remaining
```

Need to either go back to the previous `while true; ...` stuff, or else, maybe just `touch $CHIA_ROOT/log/debug.log` before trying to tail it